src/networks.js

```diff
@@ -0,0 +1,10 @@
+module.exports.getIds = () => {
```

Comment:
Two things:

1. Maybe we can also include the urls here (right now they are in `index.js`)
2. I'm not sure if this needs to be a function. Just exporting the object should be enough.

Reply:
@fvictorio 

> Maybe we can also include the urls here (right now they are in index.js)

Refactor done

> I'm not sure if this needs to be a function. Just exporting the object should be enough.

Exported the ids and the urls

Note:
Added a subcommand to show the urls